tests/processors/test_renderers.py

```diff
@@ -580,6 +580,4 @@ def test_no_exception(self, ei):
         """
         A missing exception does not blow up.
         """
-        assert {"exception": "MISSING"} == format_exc_info(
-            None, None, {"exc_info": ei}
-        )
+        assert {} == format_exc_info(None, None, {"exc_info": ei})
```

Comment:
Is this okay for you, @hynek ?  The `format_exc_info()` handled the `(None, None, None)` (even though its signature suggested that `exc_info: ExcInfo`) and returned `{"excepiton": "MISSING"}`.

Now, `(None, None, None)` will no longer be passed to it and thus there will no longer be a `MISSING`.

Reply:
@hynek ping